src/main/zapHomeFiles/hudtutorial/en_GB/PageAlerts.html

```diff
@@ -11,15 +11,15 @@ <h1>Page Alerts</H1>
 	which by default are shown on the left hand side. This allows you to just focus on the potential issues on
 	your current page without being distracted by issues on other pages. 
 	<p>
-	Alerts have an associated risk, and all alerts of with the same level of risk are shown together:
+	Alerts have an associated risk, and all alerts with the same level of risk are shown together:
 	<ul>
-	<li><img src="page-alerts-high.png"> High Level Page Alerts
```

Comment:
I think they were as is to match the label of the tools.

Reply:
I'm not set on it one way or another. I see the value in matching it, but I also see Dave's point that it's kind of redundant. 

@psiinon tie break it? :smirk:

Reply:
Should def match the UI, and that means I think we should have both "Page" and "Site" in all places. If we dont make those things clear people wont understand the difference...

Reply:
Ok, I'll revert the "Page" removals then.